Source/ui_unix/mainwindow.cpp

```diff
@@ -0,0 +1,198 @@
+#include "mainwindow.h"
+#include "settingsdialog.h"
+
+#include "openglwindow.h"
+
+#include <QFileDialog>
+#include <QTimer>
+#include <QWindow>
+
+#include "global.h"
+#include "GSH_OpenGLQt.h"
+#include "tools/PsfPlayer/Source/SH_OpenAL.h"
+#include "PreferenceDefs.h"
+
+#include "ui_mainwindow.h"
+
+MainWindow::MainWindow(QWidget *parent) :
+    QMainWindow(parent),
+    ui(new Ui::MainWindow)
+{
+    ui->setupUi(this);
+
+    openglpanel = new OpenGLWindow;
+    openglpanel->show();
```

Comment:
That line shouldn't be here. Makes the output window separate from the main window. (Didn't we remove that line before?)